lona/html/attribute_dict.py

```diff
@@ -19,6 +19,9 @@ def items(self):
 
     def pop(self, name):
         with self._node.lock:
+            if name not in self._attributes:
+                return None
```

Comment:
Does this make sense? The reason why this method raises a key error when the given name is not present in ``_attributes`` is that ``.pop()`` is supposed to to return the value of ``name`` and ``.remove()`` does not do this.

Reply:
What do you think if I change it as following?
1. `attributes.pop(name)` will raise an exception if name is unknown
2. `attributes.pop(name, default)` will not raise an exception, but return `default`
3. `del attributes[name]` will not raise an exception